django/cantusdb_project/main_app/admin.py

```diff
@@ -170,6 +171,7 @@ def get_source_siglum(self, obj):
         "source",
         "feast",
     )
+    readonly_fields = READ_ONLY + ("incipit",)
```

Comment:
This isn't exactly related, but I am having trouble following the readonly_fields assignment in this file. 

- Why does a `sequence` and a `chant` have different `readonly_fields`? 
- Most of the Admin classes use the `readonly_fields` of the `BaseModelAdmin` but it looks like `date_created` and `date_updated` are not marked as `readonly_fields` there and yet they don't show up in the admin view? 

I'll continue to poke around...but let me know if I am missing something obvious...

Reply:
Do you know why, though, on current production in many places `date_created` and `date_updated` don't show up in the edit form? For example: https://cantusdatabase.org/admin/main_app/century/3861/change/

My current guess is that the `Meta` class and the customizations (in the Century form, the `name` field, for example) are not working well together. See 

https://github.com/DDMAL/CantusDB/blob/07be678c5e18934bf958d9f3aae728ec54e3e264/django/cantusdb_project/main_app/forms.py#L488-L493

Reply:
Got it. In principle, we could open an issue saying "there's a bug: some fields that we expect to be showing up in the admin area, like `date_created` and `date_updated`, are not actually showing up". This PR would fix the lack of `date_created`/`_updated`, but since we haven't figured out why they were missing in the first place, it could be that there are other fields that are missing that we're not aware of.

Am I understanding what you're getting at?

Reply:
fwiw, with this PR as it currently stands, for all the models (other than Chant, Sequence and Source, which have tons of fields and I haven't checked them thoroughly), all the fields that we expect to appear (i.e., those defined in the model itself, plus the four defined in BaseModel) all appear as expected in the Admin area.

Reply:
> fwiw, with this PR as it currently stands, for all the models (other than Chant, Sequence and Source, which have tons of fields and I haven't checked them thoroughly), all the fields that we expect to appear (i.e., those defined in the model itself, plus the four defined in BaseModel) all appear as expected in the Admin area.

Yes, it looks good. I guess I don't know why these fields weren't showing up before, but they seem to be with this PR (I also just looked through things locally), so I think we can resolve this.